File: help/pt_pt/exercicios_da_ficha_4/Ficha04_012/Ficha04_012.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha04_012 {

    //Main Function
    static void Main() {
        Console.Write("Real 1 \t: " );
        double r1 = double.Parse(Console.ReadLine());
        Console.Write("Real 2 \t: " );
        double r2 = double.Parse(Console.ReadLine());
        Console.Write("\nOrdem decrescente\n" );
        if(r1 > r2 ){
            Console.Write(r1 + "\t" + r2 );
        }else{
            Console.Write(r2 + "\t" + r1 );
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:32

File: help/pt_pt/exercicios_da_ficha_4/Ficha04_012/Ficha04_012.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Real 1 \t: " );// print text
    r1 = parseFloat(prompt("Real 1 \t: " ,"0.0") );
    
    document.write(r1 + "\n");// print input 
    document.write("Real 2 \t: " );// print text
    r2 = parseFloat(prompt("Real 2 \t: " ,"0.0") );
    
    document.write(r2 + "\n");// print input 
    document.write("\nOrdem decrescente\n" );
    if(r1 > r2 ){
        document.write(r1 + "\t" + r2 );
    }else{
        document.write(r2 + "\t" + r1 );
    }
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:32
